src/main/scala/org/hammerlab/guacamole/reads/Read.scala

```diff
@@ -170,6 +169,7 @@ object Read extends Logging {
   def fromSAMRecordOpt(record: SAMRecord,
```

Comment:
Not strictly related to this change, but what does `Opt` mean here? Optimized? Option? When would it return `None`?